runtime/oti/cfreader.h

```diff
@@ -150,6 +150,12 @@
 #define NEXT_U32(value, index) ((value = ((U_32)(index)[0] << 24) | ((U_32)(index)[1] << 16) | ((U_32)(index)[2] << 8) | (U_32)(index)[3]), index += 4, value)
 #define NEXT_I32(value, index) ((value = (I_32) (((U_32)(index)[0] << 24) | ((U_32)(index)[1] << 16) | ((U_32)(index)[2] << 8) | (U_32)(index)[3])), index += 4, value)
 
+#define PEEK_U8(value, index) (value = *(index))
```

Comment:
Can these be used in the NEXT macros to avoid the duplication? Also, the use of `, value` is inconsistent in the macros. I believe the U8 macro above should have it to avoid possible compiler warnings.
```
#define NEXT_U16(value, index) (PEEK_U16(value, index), index += 2, value)
```